src/index.ts

```diff
@@ -88,3 +88,7 @@ export {
   // OpenApi Decorators
   OpenApiSecurityScheme
 } from "./httpServer/middleware";
+
+export {
+  KafkaConsume
```

Comment:
Not seeing the kafka decorators here.

Reply:
What do you mean?

Reply:
I think I misread the import in the test file when I thought that.  Withdrawn.